ENIGMAsystem/SHELL/Bridges/OpenGL/GLinit.cpp

```diff
@@ -0,0 +1,14 @@
+#include "GLinit.h"
+#include "Widget_Systems/widgets_mandatory.h"
+
+#include <GL/glew.h>
+
+namespace enigma {
+
+void gl_init() {

```

Comment:
Not really initializing GL; initializing its extensions. Probably worth a better name.

Reply:
What do you think about `gl_load`?

Reply:
Ok: b22a58de47e6ceec916aad2af1b21b1264554867

It's true, I prefer `gl_load` myself since that's what we usually calling this, an OpenGL _loader_.

Reply:
I was thinking more like `gl_init_exts` or `gl_load_libs`. You can be more verbose, but I'm not sure about less.

Reply:
Alright: 9a2dbe70f7447f8aac3f43e3980ed2964b1c7831

So `gl_load_exts` it is then 😛

Reply:
think he just wants gl_init_ext or w/e gl_load is same thing as gl_init

Reply:
I asked him, he said that much is fine, it was just too short. We typically refer to this as an OpenGL "loader" because of it _loads_ the function pointers from the dll.